apps/emqx_management/test/emqx_mgmt_api_clients_SUITE.erl

```diff
@@ -1535,3 +1546,30 @@ connect_client(Opts) ->
     ]),
     {ok, _} = emqtt:connect(C),
     C.
+
+%% To avoid a race condition where we try to delete the session while it's terminating and
+%% committing.  This shouldn't happen realistically, because we have a safe grace period
+%% before attempting to GC a session.
+stop_and_commit(Client) ->
+    {ok, {ok, _}} =
+        ?wait_async_action(
+            emqtt:stop(Client),
+            #{?snk_kind := persistent_session_ds_terminate}
+        ),
+    ok.
+
+delete_all_sessions(N) ->
```

Comment:
I'd definitely vote for not reaching that deep into the implementation here. Should be enough to `emqtt:disconnect(C, ?RC_SUCCESS, #{'Session-Expiry-Interval' => 0})` each client? After all, we strive to respect the spec in this regard, thereby these sessions should disappear from API responses anyway.

Reply:
:heavy_check_mark: